Makefile

```diff
@@ -86,7 +86,7 @@ define agdaToHtml
     @echo "Generating $@"
     $(AGDA_RUN) --html --html-dir $(HTML_DIR) $<
 endef
-$(HTML_DIR)/$(LEDGER).PDF.html : src/$(LEDGER)/PDF.lagda
+$(HTML_DIR)/Html.html : src/Html.agda
```

Comment:
Do we really want to call it `Html.html`?  How about `cardano-ledger.html`?  In any case, we should probably use a variable name (e.g., `LEDGER = cardano-ledger`) and then use $(LEDGER).html and src/$(LEDGER).agda (and $(LEDGER).tex in line 79).

Reply:
It makes sense. 
I went with reusing the variable LEDGER which is already defined in the Makefile